pom.xml

```diff
@@ -628,6 +628,7 @@
                         <exclude>Jenkinsfile</exclude>
                         <exclude>**/codestyle/*</exclude>
                         <exclude>**/resources/META-INF/**</exclude>
+                        <exclude>**/.tmp/**</exclude>
```

Comment:
no need to add `.tmp` dir, it's not included in maven project.

Reply:
Have removed it